learning_resources_search/views.py

```diff
@@ -63,6 +64,7 @@ class LearningResourcesSearchView(ESView):
 
     permission_classes = ()
 
+    @method_decorator(cache_page(60 * 60 * 24, cache="redis", key_prefix="search"))
```

Comment:
The length of time that the cache is kept alive worries me 24hrs is a long time- although if we clear the cache after every task that updates the index, the actual age of the caches should be less than a couple hours, probably? Also are we worried about memory usage for storing the caches?